ethsigner/src/main/java/tech/pegasys/ethsigner/jsonrpc/response/JsonRpcError.java

```diff
@@ -67,6 +67,9 @@
   NODE_WHITELIST_MISSING_ENTRY(-32000, "Cannot remove an absent node from whitelist"),
   NODE_WHITELIST_BOOTNODE_CANNOT_BE_REMOVED(-32000, "Cannot remove a bootnode from whitelist"),
 
+  // Transaction signing errors
+  SIGNING_FROM_IS_NOT_AN_UNLOCKED_ACCOUNT(-32000, "from parameter does match an unlocked account"),
```

Comment:
hmmm - should the "from" be capitialised (and/or in 'from')?

Reply:
Yes, for the consistency.

Reply:
oh, do you mean the text in the message? (not the Enum)

Reply:
In the message .... the parameter is actually lower case - but the msg looks odd...

Reply:
The parameter is lower case, it'll just be confusing to have the message use upper case

Reply:
...I could always change it from from to sender 😁

Reply:
actually - Sender is nice. The high level error message shouldn't need to track the literal definition of a transaction

Reply:
Updated message